src/goto-harness/recursive_initialization.cpp

```diff
@@ -169,11 +169,11 @@ void recursive_initializationt::initialize(
             goto_model.symbol_table.lookup_ref(fun_name);
           const auto proper_init_case = code_function_callt{
             fun_symbol.symbol_expr(), {depth, address_of_exprt{lhs}}};
-
+          const auto should_make_equal =
```

Comment:
:no_entry_sign: Looks like this is a separate case and therefore should have a separate test? :safety_pin: